README.md

```diff
@@ -624,69 +732,96 @@ This Tree-sitter query:
   (#scope_id! "tuple")
 )
 ```
-formats this piece of code:
+
+...formats this piece of code:
```

Comment:
I don't care either way, but this looks like a pretty unusual typographic style.

Reply:
I don't like the look of what appears to be a new sentence, but is actually just a follow-on clause interjected with a code block, starting with a lowercase letter. I add ellipses in this case as an indicator of this.